app-android/src/main/java/io/github/droidkaigi/confsched2023/KaigiApp.kt

```diff
@@ -150,27 +150,18 @@ private fun NavGraphBuilder.mainScreen(
             )
             nestedAboutScreen(
                 onAboutItemClick = { aboutItem ->
+                    val portalBaseUrl = if (defaultLang() == JAPANESE) {
+                        "https://portal.droidkaigi.jp"
+                    } else {
+                        "https://portal.droidkaigi.jp/en"
```

Comment:
It seems that I can't access this url. 👀 
`https://portal.droidkaigi.jp/en`

Reply:
@takahirom 
Sorry if I am wrong.
I checked the operation and it seems to work fine.
Could you please tell me how you checked it?

https://github.com/DroidKaigi/conference-app-2023/assets/13657682/92d63a86-c3de-46c9-89d5-f23d21f7e551


https://github.com/DroidKaigi/conference-app-2023/assets/13657682/44684b7f-8449-4669-b2a2-f05eca6339cc

Reply:
I opened it with Mac OS's Chrome. I'm not sure why 👀

Reply:
@takahirom 
Sorry if I have made some recognition error.🙇‍♂️
The portalBaseUrl is not to be used by itself,
The portalBaseUrl is not used by itself, but is assembled and used as a URL, such as
https://portal.droidkaigi.jp/about/code-of-conduct
https://portal.droidkaigi.jp/en/about/code-of-conduct
https://portal.droidkaigi.jp/about/privacy
https://portal.droidkaigi.jp/en/about/privacy
So I don't see any problem with this implementation.
I would appreciate it if you could describe the problem in more detail.🙇‍♂️

Reply:
Oh, sorry

Reply:
I understand what you mean! Sorry for bothering you!